custom_dc/custom_data.md

```diff
@@ -116,11 +167,209 @@ geoId/06,2021,555,666
+      "provenance": "UN_WHO",
+      "observationProperties" : {
+        "unit": "percentage"
+      }
+    }
+  },
+  "variables": {
+    "Adult_curr_cig_smokers": {
+      "name": "Adult Current Cigarette Smokers",
+      "description": "Percentage of smokers in the total adult population",
+      "searchDescriptions": [
+        "Prevalence of smoking among adults in world countries in the years 2016 - 2019."
+      ],
+      "group": "WHO"
+    },
+    "Adult_curr_cig_smokers_female": {
+      "name": "Adult Current Cigarette Smokers Female",
+      "description": "Percentage of smokers in the female adult population",
+      "searchDescriptions": [
+        "Prevalence of smoking among adult women in world countries in the years 2016 - 2019."
+      ],
+      "group": "WHO"
+    },
+      "Adult_curr_cig_smokers_male": {
+      "name": "Adult Current Cigarette Smokers Male",
+      "description": "Percentage of smokers in the male adult population",
+      "searchDescriptions": [
+        "Prevalence of smoking among adult men in world countries in the years 2016 - 2019."
+      ],
+      "group": "WHO"
+    }
+  },
+ "sources": {
+    "custom.who.int": {
+      "url": "https://custom.who.int",
+      "provenances": {
+        "UN_WHO": "https://custom.who.int/data/gho/indicator-metadata-registry/imr-details/6128"
+      }
+    }
+  }
+}
+```
+The following fields are specific to the variable-per-column format:
+
+- `input_files`:
+  - `entityType`: This must be an existing entity class in the Data Commons knowledge graph; it's most commonly a [place type](/place_types.html).
+- `variables`: This section is optional but recommended. You can use it to override names and associate additional properties with the statistical variables in the files, using the parameters described below. All parameters are optional.
+  - `name`: A human-friendly readable name that will be shown throughout the UI.
+  - `description`: A more detailed name that will be shown in the Statistical Variable Explorer.
+  - `searchDescriptions`: This is a comma-separated list of natural-language text descriptions of the variable; these descriptions will be used to generate embeddings for the NL query interface.
+  - `group`: This will display the variables as a group in the Statistical Variable Explorer, using the name you provide as heading. 
+
+The other fields are explained in the [Data config file specification reference](#json-ref).
+
+## Prepare your data using explicit schema
+
+In this section, we will walk you through a concrete example of how to go about setting up your CSV, MCF and JSON files.
+
+### Write the MCF file {#mcf}
+
+Nodes in the Data Commons knowledge graph are defined in Metadata Content Format (MCF). For custom Data Commons using explicit schema, you must define your statistical variables using MCF. Here's an example of defining the same statistical variables in the WHO data in MCF:
+
+```
+Node: dcid:Adult_curr_cig_smokers
```

Comment:
Can this be Camel_Case with capitals?
`Adult_Curr_Cig_Smokers`

Reply:
This is actually exactly how they are doing it in UN!  Should we do it that way to illustrate a good practice? I can do that.